recipes/centos_vertx/cccp.yml

```diff
@@ -0,0 +1 @@
+job-id: vertx
```

Comment:
yes but is it required in each folder or for example at the root of the repository ?